plugin.json

```diff
@@ -10,4 +10,6 @@
         { "hook": "filter:navigation.available", "method": "getNavigation"}
     ],
     "templates": "static/templates"
-}
\ No newline at end of file
+    "languages": "languages",
+	"defaultLang": "en_GB"
```

Comment:
Adjust indentation to match please :smile: 

Can you merge all changes into a single pull request?